modules/cluster/cluster.tf

```diff
@@ -16,7 +16,7 @@ resource "oci_containerengine_cluster" "k8s_cluster" {
   }
 
   endpoint_config {
-    is_public_ip_enabled = var.control_plane_is_public
+    is_public_ip_enabled = var.control_plane_is_public && var.assign_public_ip_to_control_planeßß
```

Comment:
you have extra characters at the end of this line.

Reply:
@hyder - Done